contracts/delegatecall/LibEtherForENS.sol

```diff
@@ -0,0 +1,25 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.20;
+
+abstract contract Resolver {
+    function addr(bytes32 node) public view virtual returns (address);
+}
+
+abstract contract ENS {
+    function resolver(bytes32 node) public view virtual returns (Resolver);
+}
```

Comment:
Maybe we could add `@ensdomains/ens-contracts` as a dependency and import these definitions from there?